framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/builder/ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java

```diff
@@ -87,7 +94,11 @@ public ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl() {
 
     @Override
     public ClusteredWorkbenchKieServerDatabasePersistentScenario build() {
-        return new ClusteredWorkbenchKieServerDatabasePersistentScenarioImpl(kieApp, deploySSO);
+        ClusteredWorkbenchKieServerDatabasePersistentScenarioImpl scenario = new ClusteredWorkbenchKieServerDatabasePersistentScenarioImpl(kieApp, deploySSO);
+        if (deployInternalMaven) {
+            scenario.addExtraDeployment(extraDeploymentFactory.get(MavenRepositoryExternalDeployment.ID, new HashMap<String, String>()));
```

Comment:
The naming is a bit confusing here. There is used MavenRepositoryExternalDeployment in case deployInternalMaven is true. Reader could be confused if external Maven repo is initialized here or internal.

Reply:
updated it. Let me know if it is better.

Reply:
Looks better now